shared/src/commonTest/kotlin/com/mbta/tid/mbta_app/viewModel/FavoritesViewModelTest.kt

```diff
@@ -478,17 +504,21 @@ class FavoritesViewModelTest : KoinTest {
                 )
             }
 
-        setUpKoin(objects)
+        val dispatcher = StandardTestDispatcher(testScheduler)
+        setUpKoin(objects, dispatcher)
 
         val viewModel: FavoritesViewModel = get()
         viewModel.setAlerts(AlertsStreamDataResponse(emptyMap()))
         viewModel.setNow(now)
         viewModel.setLocation(stop1.position)
 
         testViewModelFlow(viewModel).test {
+            println("before advance")
+            advanceUntilIdle()
+            println("after advance")
             assertEquals(
                 emptyList(),
-                awaitItemSatisfying { it.routeCardData != null }
+                expectMostRecentItem()
                     .routeCardData!!
```

Comment:
This is still flaking 🤔 I'm not sure how `expectMostRecentItem` here could *not* have routeCardData set.

Reply:
Maybe the `advanceUntilIdle` sets the stop IDs but a second `advanceUntilIdle` would be necessary to fetch the predictions?